main/guides/getting-started/syncing-up.md

```diff
@@ -0,0 +1,86 @@
+|         | agoric-upgrade-5     |     3,565,000  | a2ebcdb21418bb157f9c747a042b2a859b2a5986 |    ag0     |             N/A            |
+|         | agoric-upgrade-6     |     5,901,622  | 31c78ba3aa872b54c4de448763c5b8044b8f950c |    ag0     |             N/A            |
+|         | agoric-upgrade-7-2   |     6,263,783  | f4759c9f15b869c453f847a63ba734cacb9a991a |    ag0     |             N/A            |
+|         | agoric-upgrade-8     |     7,179,262  | 2c812d22161cd297587979b262eab6e2cc76e23d |    agd     |    agoric/agoric-sdk:29    |
+| Current | agoric-upgrade-8-1   |     7,179,262  | 08ca9d4fd8413da59b73d53e12851fe00583ddc1 |    agd     |    agoric/agoric-sdk:30    |
+
+
+### agoric-3.1
+
+1. `git clone https://github.com/Agoric/ag0.git && cd ag0`
+1. `git checkout tags/agoric-3.1`
+1. `make install`
+1. `ag0 init follow --chain-id agoric-3`
+1. `curl https://main.rpc.agoric.net/genesis | jq .result.genesis > $HOME/.agoric/config/genesis.json`
+1. `ag0 start`
+
+   chain will halt at 3,565,000
+
+### agoric-upgrade-5
+
+1. `git clone https://github.com/Agoric/ag0.git; cd ag0 && git checkout tags/agoric-upgrade-5`
+1. `make install`
+1. `ag0 start`
+
+   chain will halt at 5,901,622
+
+### agoric-upgrade-6
+
+1. `git clone https://github.com/Agoric/ag0.git; cd ag0 && git checkout tags/agoric-upgrade-6`
+1. `make install`
+1. `ag0 start`
+
+   chain will halt at 6,263,783
+
+### agoric-upgrade-7-2
+
+agoric-upgrade-7-2 is a agoric-upgrade-7 compatible soft-patch, and can be used in place of agoric-upgrade-7.  This soft-patch includes the dragonberry security updates.
+
+1. `git clone https://github.com/Agoric/ag0.git; cd ag0 && git checkout tags/agoric-upgrade-7-2`
+1. `make install`
+1. `ag0 start`
+
+   chain will halt at 7,179,262
+
+### note about ag0 -> agd
+
+The `agd` upgrade starts now, and you need to ensure you have
+nodejs 16+ LTS installed, along with any dependencies mentioned 
+in https://github.com/Agoric/agoric-sdk/blob/master/README.md
+
+This guide assumes you've git cloned the agoric-sdk from 
+https://github.com/Agoric/agoric-sdk/ to a folder 
+named `agoric-sdk` and you've installed the required dependencies 
+noted in the readme.
+
+### agoric-upgrade-8
+
+1. `git clone https://github.com/agoric/agoric-sdk.git; cd agoric-sdk && git checkout tags/agoric-upgrade-8`
+1. `yarn install && yarn build`
+1. `agd start`
+
+After more than 10 blocks have successfully synced, stop the `agd` process 
+and update to `agoric-upgrade-8-1`, a soft upgrade.  This soft upgrade includes 
+logic to handle a memory condition that may lead to your node falling out of consensus. 
+It is recommended to upgrade to `agoric-upgrade-8-1` as soon as you have successfully 
+produced a handful of blocks with the `agoric-upgrade-8` upgrade.
+
+### agoric-upgrade-8-1
+#### Current Community Snapshot
+
+1. `git clone https://github.com/agoric/agoric-sdk.git;  agoric-sdk && git checkout tags/agoric-upgrade-8-1`
```

Comment:
I was going to ask if we're comfortable giving instructions on using a community snapshot without "use at your own risk" warnings... but this doesn't seem to include instructions about getting a community snapshot.

Reply:
We have no alternatives to offer users besides our GitHub as a source repo, so how would a user make an informed risk decision worth highlighting here?

Reply:
I guess I don't understand what this section is for. I thought it was about downloading a snapshot and resuming from there. If somebody follows the directions in this section, I don't understand what they should expect to get as a result.

We could point them at https://github.com/Agoric/validator-profiles/wiki/Snapshots , which has a **Use at your own risk.** disclaimer at the top. As noted there, discussion of this topic goes back to Feb in https://github.com/Agoric/testnet-notes/issues/42 . Or we could migrate that info to community.agoric.com ... I gather that in the cosmos community, sharing validator profiles and "look what I did for the community" is more traditionally done in discourse forums than github wikis. We could start a thread where we don't give any snapshot pointers, but we invite snapshot providers to give pointers.

Reply:
This is "how to run the most recent version of the code with the latest upgrade" if you bring along current state from a snapshot